app/Http/Controllers/BookController.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Book;
+use App\Models\Category;
+use Illuminate\Http\Request;
+
+class BookController extends Controller
+{
+    public function index(Request $request)
+    {
+        $page = $request->page;
+        $books = Book::with('categories')->orderBy('id', 'desc')->where('status', config('book.visible'))->paginate(config('pagination.limit'));
+
+        return view('client.home', compact('books', 'page'));
+    }
+
+    public function cateBook($categoryId)
+    {
+        $category = Category::findOrFail($categoryId)->load('books');
+
+        return view('client.category_book', compact('category'));
+    }
+
+    public function detail($id)
```

Comment:
```suggestion
    public function getDetailBook($id)
```
function phải là động từ